hail/python/hailtop/batch/docs/api.rst

```diff
@@ -58,6 +58,22 @@ share a common root, but each file has its own extension.
     resource.ResourceGroup
 
 
+Batch Pool Executor
+~~~~~~~~~~~~~~~~~~~
+
+A :class:`.BatchPoolExecutor` provides roughly the same interface as the Python
+standard library's :class:`.concurrent.futures.Executor`. It facilitates
+executing arbitrary python functions in the cloud.
+
+.. autosummary::
+    :toctree: api/batch/
+    :nosignatures:
+    :template: class.rst
+
+    batch.Batch
```

Comment:
This isn't the right thing. I think you need batch_pool_executor.BatchPoolExecutor.